### PR TITLE
Bump version of Connect-JS

### PR DESCRIPTION
**Pull request by riquelme-stripe**
Bumped version to fix dependency issues.

### CODE REVIEW

**Review by jorgea-stripe — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stripe/react-connect-js",
-  "version": "3.3.9",
+  "version": "3.3.10",
```
Comment (jorgea-stripe) [inline package.json:3]: Why is this bumped in this PR? This should be bumped during publishing only

File: package.json [hunk 2]
```diff
@@ -84,7 +84,7 @@
     "zx": "^4.2.0"
   },
   "peerDependencies": {
-    "@stripe/connect-js": ">=3.3.9",
+    "@stripe/connect-js": ">=3.3.10",
```
Comment (jorgea-stripe) [inline package.json:87]: If we bump this, we also need to bump `dependencies`

**Review by jorgea-stripe — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (riquelme-stripe) [reply package.json:3]: This should have been bumped last time it seems. React-Connect-JS is currently published as 3.3.10.

**Comment by riquelme-stripe** [comment]
Joanna thinks the issue is something else. Closing

File: package.json (hunk 1 above)
Comment (jorgea-stripe) [reply package.json:3]: Let's bump it in a separate PR

**Pull request closed without merging**